corehq/motech/views.py

```diff
@@ -101,21 +101,15 @@ def page_url(self):
         return reverse(self.urlname, args=[self.domain, pk])
 
 
+@method_decorator(requires_privilege_with_fallback(privileges.DATA_FORWARDING),
+                  name='dispatch')
 @method_decorator(require_permission(Permissions.edit_motech), name='dispatch')
 class ConnectionSettingsView(BaseProjectSettingsView, FormView):
     urlname = 'connection_settings_view'
     page_title = ugettext_lazy('Connection Settings')
     template_name = 'motech/connection_settings.html'
     form_class = ConnectionSettingsFormSet  # NOTE: form_class is a formset
 
-    def dispatch(self, request, *args, **kwargs):
-        if not (
-                toggles.DHIS2_INTEGRATION.enabled_for_request(request)
-                or toggles.INCREMENTAL_EXPORTS.enabled_for_request(request)
```

Comment:
will any of these domains need their privileges updated to access the DATA_FORWARDING permission?